docs/collecting-data/collecting-from-own-applications/python-tracker/tracking-specific-events/index.md

```diff
@@ -1,528 +1,282 @@
-```
-
-#### 3. Event subject (since v0.9.0)
-
-Since version 0.9.0, it is possible to set the Subject per-event, in order to augment the event with extra information without having to change the Subject at the Tracker level. This provides a thread safe way to track multiple subjects.
-
-This is supported as an optional keyword argument by all track methods. For example:
+## Out-of-the-box Events
+The Python tracker provides classes for tracking different types of events. They are listed below:
 
-```python
-
-evSubject = Subject().set_user_id("1234")
-t.track_page_view("www.example.com", event_subject=evSubject)
-```
+| **Event** | **Description** |
+| --- | --- |
+| `SelfDescribing` | Track custom events with custom schemas |
+| `PageView` | Track views of web pages |
+| `PagePing` | Track engagement on web pages over time |
+| `ScreenView` | Track views of a screen |
+| `StructuredEvent` | Track custom events without schemas |
 
-### Tracker method return values
+## Creating a custom event (`SelfDescribing`)
+To track data using a `SelfDescribing` event, the data must be structured as a `SelfDescribingJson` object. These require two fields, a URI for a self-describing JSON schema and the data in the form of a `PayloadDict`. The data must be valid against the schema. 
 
-All tracker methods will return the tracker instance, allowing tracker methods to be chained:
+A simple initialisation for a link click event looks like this:
 
 ```python
-e = AsyncEmitter("d3rkrsqld9gmqf.cloudfront.net")
-t = Tracker(e)
-
-t.track_page_view("http://www.example.com").track_screen_view("title screen")x
+link_click = SelfDescribing(
+  SelfDescribingJson(
+    "iglu:com.snowplowanalytics.snowplow/link_click/jsonschema/1-0-1",
+    {"targetUrl": "https://www.snowplow.io"},
+  ),
+)
+t.track(link_click)
 ```
 
-### Track self-describing event
-
-Use `track_self_describing_event()` to track an event types that you have defined yourself.
+## Creating a `PageView` Event
+Track page views with the `PageView` event. 
 
-This method's arguments are:
-
-| **Argument** | **Description** | **Required?** | **Type** |
+| **Property** | **Description** | **Type** | **Required?** |
 | --- | --- | --- | --- |
-| `event_json` | The properties of the event | Yes | SelfDescribingJson |
-| `context` | Custom context for the event | No | List(SelfDescribingJson) |
-| `tstamp` | When the unstructured event occurred | No | Positive integer |
-| `event_subject` (since v0.9.0) | The subject for the event | No | Subject |
+| `page_url` | URL of the viewed page | `string` | Yes |
+| `page_title` | Title of the viewed page | `string` | No |
+| `referrer` | The address which linked to the page | `string` | No |
+| `event_subject` | The subject for the event | `Subject` | No |
+| `context` | Custom context for the event | `List(SelfDescribingJson)` | No |
+| `true_timestamp` | When the page view occurred | `int | float` | No |
```

Comment:
The | in `int | float` is understood by Docusaurus as another column so it's not rendered correctly.

```suggestion
| `true_timestamp` | When the page view occurred | `int or float` | No |
```